cmd/osm-controller/main.go

```diff
@@ -150,3 +158,12 @@ func parseClusterDNSIPs(s string) ([]net.IP, error) {
 	}
 	return ips, nil
 }
+
+// getKubeConfigPath returns the path to the kubeconfig file.
+func getKubeConfigPath() string {
```

Comment:
This defaulting is already internally implemented by the controller-runtime package, as you can see here: https://github.com/kubernetes-sigs/controller-runtime/blob/619e6b92dab99f0b6c132d0223e8dd9efa1fb1e9/pkg/client/config/config.go#L43-L61
Do we really need something like this?

Reply:
Yes i have seen that but the thing is we need the `path` to kubeconfig file here. Not the actual configuration.